funnel/forms/participant.py

```diff
@@ -10,11 +10,21 @@
 class ParticipantForm(forms.Form):
     fullname = forms.StringField(__("Fullname"), validators=[forms.validators.DataRequired()])
     email = forms.EmailField(__("Email"), validators=[forms.validators.DataRequired(), forms.validators.Length(max=80)])
-    phone = forms.StringField(__("Phone number"), validators=[forms.validators.Length(max=80)])
-    city = forms.StringField(__("City"), validators=[forms.validators.Length(max=80)])
-    company = forms.StringField(__("Company"), validators=[forms.validators.Length(max=80)])
-    job_title = forms.StringField(__("Job title"), validators=[forms.validators.Length(max=80)])
-    twitter = forms.StringField(__("Twitter"), validators=[forms.validators.Length(max=15)])
+    phone = forms.StringField(__("Phone number"),
+        description=__("Optional"),
```

Comment:
In these fields we need to explain why they are being asked. Just "Optional" isn't enough.